Odata-docs/client/using-dataservice-ihttpclientfactory.md

```diff
@@ -0,0 +1,82 @@
+# Using DataserviceContext IHttpClientFactory in Microsoft.OData.Client
+
+Ever wanted to use `HttpClient` with `Microsoft.OData.Client`? Now you can! In this post, we'll show you how to do it easily and highlight the benefits.
+
+## Why Use IHttpClientFactory?
+
+IHttpClientFactory is a feature in .NET Core that helps you manage HttpClient instances. Here are some benefits:
```

Comment:
```suggestion
`IHttpClientFactory` is a feature in .NET Core that helps you manage `HttpClient` instances. Here are some benefits:
```

Format references to library APIs as code. Also consider making them links to the appropriate .NET reference docs where possible, especially when mentioning the first time, e.g.:

```markdown
[`IHttpClientFactory`](https://learn.microsoft.com/dotnet/core/extensions/httpclient-factory)
```

Ensure to remove `en-us` and version monikors from the URLs to make them more portable for different user default locales.

Reply:
Resolved.